src/web/src/components/Posts/Users.tsx

```diff
@@ -0,0 +1,76 @@
+import { VscGithub } from 'react-icons/vsc';
+import { createStyles, makeStyles, Theme } from '@material-ui/core';
+import TelescopeAvatar from '../TelescopeAvatar';
+
+const useStyles = makeStyles((theme: Theme) =>
+  createStyles({
+    link: {
+      textDecoration: 'none',
+      color: theme.palette.text.primary,
+      '&:hover': {
+        textDecorationLine: 'underline',
+      },
+    },
+    GitHubInfo: {
+      lineHeight: 'normal',
+      fontSize: '1.2rem',
+    },
+    GitHubLinkTitle: {
+      fontSize: '1.4rem',
+      margin: 0,
+      paddingTop: '1rem',
+    },
+    icon: {
+      fontSize: '2rem',
+      marginRight: '1rem',
+      verticalAlign: 'text-bottom',
+    },
+    users: {
+      paddingLeft: 0,
+      display: 'flex',
+      flexWrap: 'wrap',
+      gap: '1.5rem',
+    },
+    user: {
+      listStyle: 'none',
+    },
+  })
+);
+
+const getUserImage = (user: string) => {
```

Comment:
Two things:

1. If your function returns the value of an expression, you don't need `{ return ... }` and can simply do `=> expr`:

```ts
const getUserImage = (user: string) => `https://github.com/${user}.png`
```

2. Add an argument here to accept a size value, and include it in the URL so we don't load an image larger than we need: `https://github.com/${user}/png?size={size}`.  You can default it to `25`.

```ts
const getUserImage = (user: string, size: number = 25) => `https://github.com/${user}.png?size={size}`
```

Reply:
I played around with this a bit. 
I wanted to make the code can be reused elsewhere, where the size may change. I made the avatarsize prop for the whole component, so we will pull the image from github in that size, but also pass that same size to the Telescope avatar. the prop avatarsize is optional, but the default will be 25.